templates/partial/navigation-canonical.html

```diff
@@ -0,0 +1,193 @@
+          <div class="p-media-object__details">
+            <h3 class="p-media-object__title"><a href="https://maas.io/">MAAS&nbsp;&rsaquo;</a></h3>
```

Comment:
Looking at [headingsMap](https://chrome.google.com/webstore/detail/headingsmap/flbjommegcjonpdmenkdiocclhjacmbi?hl=en), I think we should move away from including heading elements in this dropdown navigation - the heading levels seem more related to the style associated with them than their semantic position in the hierarchy of the page, so it might be better to replace the heading elements in this file with, for example:

```suggestion
            <span class="p-media-object__title p-heading--3"><a href="https://maas.io/">MAAS&nbsp;&rsaquo;</a></span>
```

I'm gonna mention this in the guild channel too, I can see why heading levels seem useful in navigation.